packages/flutter_robusta/test/src/stub.dart

```diff
@@ -34,4 +31,8 @@ class TestScreen extends Screen implements InitialScreen {
 
   @override
   String get routePath => '/test';
+
+  /// Checks for MaterialApp in the widget tree.
```

Comment:
Shouldn't be `Cupertino App` instead of `Material App` ?

Reply:
Fixed! Thank you.